### PR TITLE
Request to merge feature menu

### PR DESCRIPTION
**Pull request by Kingstalux**
Implementing the mobile menu using recently acquired JavaScript knowledge.
Making sure that when the hamburger icon is clicked the menu appears.
When the x-icon is clicked the menu disappears
When any of the links  on the menu are clicked, the menu disappears and you are taken to the corresponding page

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -24,19 +24,30 @@
...
+
+      
+      <div id="drop-menu" class="menu">
+        <img onclick="myFunction1()" src="images/Iconx.png" alt="cancel" class="cancel">
+        <ul>
+          <li><a href="#Portfolio" onclick="myFunction1()">Portfolio</a><br><hr></li>
+          <li><a href="#About" onclick="myFunction1()">About</a><br><hr></li>
+          <li><a href="#Contacts" onclick="myFunction1()">Contacts</a><br><hr></li>
+        </ul>
+      </div>
```
Comment (rukundoeric) [inline index.html:36]: - [x] <b>REQUIRED:</b> When I click on hamburger menu button, nothing is happening. kindly fix it.

File: script.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+function myFunction() {
+  document.getElementById('drop-menu').classList.add('show');
+}
+
+function myFunction1() {
+  document.getElementById('drop-menu').classList.remove('show');
+}
```
Comment (rukundoeric) [inline script.js]: - [x] <b>REQUIRED:<b/> Prefer using ES6 arrow function. Actually, there is nothing wrong with ES5 functions syntax but it is mandatory to use es6 features in the javascript module. besides, Arrow functions provide a simplified structure, allowing for a more concise way to write function expressions. which makes them better than ES5 functions(function declarations). Here is the syntax of an ES6 arrow function...
```javascript
  const functionName = () => {
     // code here
  }
```

**Review by rukundoeric — CHANGES_REQUESTED** [review]
Hi @Kingstalux, 

_Great job so far 👏 💯 👍. However, there are few issues we need to address before we move on. Please, check the comments below._
## Highlights
  - Follows git workflow.  💯
  - Codebase looks clean and professional. 💯 

## Required Changes ♻️
  - [x] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
  - [x] Check the comments under the review.

Cheers and Happy coding!👏👏👏

__Feel free to leave any questions or comments in the PR thread if something is not 100% clear.__

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
> As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html (hunk 1 above)
Comment (Kingstalux) [reply index.html:36]: There seems to be no problem here, so I think the issue may be from your side. Please kindly double check to see.

File: script.js (hunk 2 above)
Comment (Kingstalux) [reply script.js]: Thanks this was helpful.

File: index.html (hunk 1 above)
Comment (rloterh) [reply index.html:36]: This issue is well resolved! ✔️

File: script.js (hunk 2 above)
Comment (rloterh) [reply script.js]: This issue is well resolved! ✔️

File: index.html [hunk 3]
```diff
@@ -24,19 +24,30 @@
 <body>
    <section class="body">
     <section class="sect">
+
+
```
Comment (rloterh) [inline index.html:28]: - [x] OPTIONAL
Please do not leave any empty lines which do not serve any purpose in your code, to enhance a cleaner code

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi Kingsely, 

Your project is complete! Great job for making the necessary changes recommended by the previous reviewer. There is nothing else to say other than... it's time to merge it 🍾 🍾 :shipit: .
Congratulations! 🎉 🎉 🥂

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

_Kindly consider my comment(s) marked [OPTIONAL] under this review:_

<br>
Cheers and Happy coding!👏 💻 🍷 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 1 more limited review per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 3 above)
Comment (Kingstalux) [reply index.html:28]: okay thanks

**Pull request merged**